src/modules/twinkleprod.js

```diff
@@ -311,18 +311,18 @@ Twinkle.prod.callbacks = {
 				params.logEndorsing = true;
 			}
 
-			// curate/patrol the page
-			if (Twinkle.getPref('markProdPagesAsPatrolled')) {
-				pageobj.triage();
-			}
-
 			pageobj.setPageText(text);
 			pageobj.setEditSummary(summaryText);
 			pageobj.setChangeTags(Twinkle.changeTags);
 			pageobj.setWatchlist(Twinkle.getPref('watchProdPages'));
 			pageobj.setCreateOption('nocreate');
 			pageobj.save(def.resolve, def.reject);
 
+			// curate/patrol the page
+			if (Twinkle.getPref('markProdPagesAsPatrolled')) {
+				pageobj.triage();
+			}
```

Comment:
This block is moved down so that by this time `.setChangeTags()` has been called on `pageobj`.